src/old/old_states/store/store.js

```diff
@@ -0,0 +1,19 @@
+import { configureStore } from '@reduxjs/toolkit';
+import todoReducer from '../slice/todoSlice';
+import storage from 'redux-persist/lib/storage';
+import { persistReducer, persistStore } from 'redux-persist';
+
+const persistConfig = {
+    key: 'root',
+    storage,
+}
+
+const persistedReducer = persistReducer(persistConfig, todoReducer)
+
+export const store = configureStore({
+    reducer: {
```

Comment:
replace with reducer: persistedReducer and fix the breaking effect where ever it is used

Reply:
can we connect for this?